app/components/form.tsx

```diff
@@ -41,7 +41,7 @@ export const Label = ({
     htmlFor={htmlFor}
     variant={smaller ? "caption" : "body1"}
     display="flex"
-    sx={{ mb: 1, color: "grey.600" }}
+    sx={{ color: "grey.600" }}
```

Comment:
Isn't color='textSecondary' sufficient ?

Reply:
It is, good catch, thanks!